lib/osf-components/addon/components/search-result-card/styles.scss

```diff
@@ -1,6 +1,7 @@
 .result-card-container {
     background-color: $color-bg-white;
     margin: 35px 35px 0;
+    padding: 10px;
     width: 60vw;
 
     dt,
```

Comment:
There's some dt and dd styling here that is no longer needed with the changes added here, so can we remove the `margin-left: 15px;` here?

Reply:
Let me check, testing now.

Reply:
@futa-ikeda the left margin for data terms and definitions is now removed.